rollup/fees/rollup_fee.go

```diff
@@ -126,25 +134,45 @@ func rlpEncode(tx *types.Transaction) ([]byte, error) {
 	return raw.Bytes(), nil
 }
 
-func readGPOStorageSlots(addr common.Address, state StateDB) (*big.Int, *big.Int, *big.Int) {
+func readGPOStorageSlots(addr common.Address, state StateDB) (*big.Int, *big.Int, *big.Int, *big.Int, *big.Int, *big.Int) {
```

Comment:
nit: a custom return type instead of 6 big.Ints might look nicer :D

Reply:
Yeah good point, added in [63c9bd9](https://github.com/scroll-tech/go-ethereum/pull/755/commits/63c9bd91990dabed33066a218f24a98d638e1996).